project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_wiki/includes/c4m_features_og_wiki.theme.inc

```diff
@@ -29,7 +28,7 @@ function theme_c4m_features_og_wiki_wiki(array $variables) {
 
     // Add wiki page button when user has permissions to add a wiki page.
     if ($group = og_context()) {
-      if (og_user_access('node', $group['gid'], 'administer group', $user)) {
+      if (node_access('create', 'wiki_page')) {
```

Comment:
This translates to 'current user can create a wiki page'.
As long as node_access() works properly, this should be fine.